tests/integration/components/polaris-layout-test.js

```diff
@@ -22,6 +22,13 @@ const layoutAnnotationWrapperSelector = buildNestedSelector(
   'div.Polaris-Layout__AnnotationWrapper'
 );
 
+const annotationSectionDescriptionSelector = (containerSelector) =>
```

Comment:
Nit: the name `annotationSectionDescriptionSelector` makes this sound like an actual selector, rather than a function that generates a selector.

Reply:
Would converting this file/component to use the new test selectors help here?

Reply:
ah yes, didn't realize we had that here already 🙂